textattack/commands/eval_model/eval_model_command.py

```diff
@@ -32,7 +32,7 @@ class EvalModelCommand(TextAttackCommand):
 
     def get_preds(self, model, inputs):
         with torch.no_grad():
-            preds = textattack.shared.utils.batch_model_predict(model, inputs)
+            preds = model(inputs)
```

Comment:
Is there any reason why you changed this line? 

You're correct that `inputs` here is already batched according to `args.batch_size`, but I think we want to keep some of the [logic from batch_model_predict](https://github.com/QData/TextAttack/blob/master/textattack/shared/utils/tensor.py#L15-L27). 

I think the best way to do it would be to remove the loop in `eval` and just get all the predictions at once using:
```python
with torch.no_grad():
  preds = textattack.shared.utils.batch_model_predict(model, inputs, batch_size=args.batch_size)
```

but there's no reason you should have to change that in your PR.

Reply:
Seems like a totally separate discussion. Reverted.